rca/landingpages/models.py

```diff
@@ -560,9 +562,61 @@ class Meta:
         verbose_name = "Landing Page - Enterprise"
 
 
+class EELandingPageRelatedEditorialPage(RelatedPage):
+    source_page = ParentalKey(
+        "landingpages.EELandingPage", related_name="related_editorial_pages"
+    )
+    panels = [PageChooserPanel("page", ["editorial.EditorialPage"])]
```

Comment:
**Question** Would it be worth being specific with the page type to aid later queries?

Reply:
I'm not sure what you mean by this?

Reply:
`RelatedPage.page` is a FK to `wagtailcore.Page`, if these pages will always be `EditorialPage`, might it be worth having a FK to `EditorialPage` instead, to make queries easier?

(feel free to ignore this comment :-) )